portality/tasks/consumer_long_running.py

```diff
@@ -2,15 +2,17 @@
 # It changes the logging configuration. If it's imported anywhere else in the app,
 # it will change the logging configuration for the entire app.
 import logging
+
 logging.getLogger("requests").setLevel(logging.WARNING)
 logging.getLogger("urllib3").setLevel(logging.WARNING)
 
 # import the queues
-from portality.tasks.redis_huey import long_running
+from portality.tasks.redis_huey import long_running  # noqa
```

Comment:
All of the imports now have this `# noqa` comment, but I'm not sure what it means

Reply:
in pycharm, developer can use `Ctrl+Alt+o` (Optimize imports)  to clean up unused import statement .

using `# noqa` can let pychram avoid cleanup and don't show warning for this import statement.

ref: 
* https://stackoverflow.com/questions/45346575/what-does-noqa-mean-in-python-comments
* https://www.jetbrains.com/help/pycharm/disabling-and-enabling-inspections.html#suppress-inspections